description: Disable --detect-odr-violations as it causes builds to fail on 64-bit Debian Jessie

--- a/build/common.gypi
+++ b/build/common.gypi
@@ -4522,7 +4522,6 @@
                   # TODO(thestig): Enable this for disabled cases.
                   [ 'buildtype!="Official" and chromeos==0 and release_valgrind_build==0 and asan==0 and lsan==0 and tsan==0 and msan==0 and ubsan==0 and ubsan_vptr==0', {
                     'ldflags': [
-                      '-Wl,--detect-odr-violations',
                     ],
                   }],
                 ],

